src/Core/src/Handlers/Window/WindowHandler.iOS.cs

```diff
@@ -1,15 +1,20 @@
 ﻿using System;
+using CoreGraphics;
+using Foundation;
 using ObjCRuntime;
 using UIKit;
 
 namespace Microsoft.Maui.Handlers
 {
 	public partial class WindowHandler : ElementHandler<IWindow, UIWindow>
 	{
+		IDisposable? _frameObserver;
+		
 		protected override void ConnectHandler(UIWindow platformView)
 		{
 			base.ConnectHandler(platformView);
-
+			// For Size Change Events on the given window.
+			_frameObserver = platformView.AddObserver("frame", Foundation.NSKeyValueObservingOptions.New, FrameAction);
```

Comment:
When do we ever remove the observer? There aren't any tests that check for memory leaks on `Window` yet.

If you put it in `DisconnectHandler`, we'd also want to check that actually gets called for `Window`'s.

Reply:
As I wrote in the description, I didn't know where to remove it. If putting it in `DisconnectHandler` is fine then I'm happy to put it there, and I can verify that it does or doesn't get called.